src/stories/Library/search-result-item/SearchResultItem.stories.tsx

```diff
@@ -47,9 +48,12 @@ export default {
   },
 } as ComponentMeta<typeof SearchResultItem>;
 
-const Template: ComponentStory<typeof SearchResultItem> = (args) => {
+export const Item: ComponentStory<typeof SearchResultItem> = (args) => {
   return <SearchResultItem {...args} />;
 };
 
-export const Item = Template.bind({});
-Item.args = {};
+export const SkeletonItem: ComponentStory<
```

Comment:
great that you added a new story for the search result item.

Reply:
;)